inc/cleantalk-public-integrations.php

```diff
@@ -2082,6 +2082,37 @@ function apbct_form__contactForm7__changeMailNotification($component)
     return (array)$component;
 }
 
+/**
+ * Test Mailoptin subscribe form for spam
+ *
+ * @return void
+ * @global State $apbct
+ */
+function apbct_form__mo_subscribe_to_email_list__testSpam()
+{
+    $checkjs = apbct_js_test(Cookie::get('ct_checkjs'));
```

Comment:
Getting JS-code there is in `base_call()` function. This is redundant here.

Reply:
Done.